remote_attestation_sessions/src/lib.rs

```diff
@@ -16,129 +16,13 @@
 
 //! Logic for performing remote attestation in multiple sessions
 
-#![no_std]
+// The client logic requires std.
+#![cfg_attr(not(feature = "client"), no_std)]
```

Comment:
Although I think it is fine for now, it would be nice if this is split further into client, server and shared crates as part of future refactoring. That refactoring would also hopefully mean that the untrusted loader would not have to have a dependency on the remote attestation and crypto crates.

Reply:
Agreed, updated